vermeer-python-client/src/pyvermeer/utils/vermeer_config.py

```
@@ -0,0 +1,37 @@
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+
+class VermeerConfig:
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)